desktop-widgets/divelistview.cpp

```diff
@@ -414,76 +414,6 @@ void DiveListView::currentChanged(const QModelIndex &current, const QModelIndex&
 	scrollTo(current);
 }
 
-void DiveListView::mouseReleaseEvent(QMouseEvent *event)
```

Comment:
typo in the commit message (selection: send selection signal from selectionChanged() -- last line "som" vs "some")

Reply:
fixed.